components/Handlers.js

```diff
@@ -0,0 +1,47 @@
+import { render } from '@asyncapi/generator-react-sdk';
+import { pascalCase } from './common';
+
+
+let subscriptionFunction = (channelName, operation, message) => `
+// subscription handler for ${channelName}        
```

Comment:
Go doc tells you documentation for functions should start with the name of the function. For example:

```go
// OnLightMeasured is the subscription handler for ${channelName}.
func OnLightMeasured (msg *message.Message) error {}
```

This applies to all the comments you wrote on the generated code. Also to any type, not only functions.